index.html

```diff
@@ -64,7 +64,7 @@ <h1>Dental Clinic</h1>
         <br>Honolulu HI 96816
       </p>
     </div>
-    <img src="images/dentalheaderimage.svg" alt="" id="header-img">
+    <img src="images/dentalheaderimage.svg" alt="dental firm header image" id="header-img">
```

Comment:
Awesome job, feel free to treat your alt attributes like a sentence, beginning with a capital letter.